### ISSUE TITLE
Add links from principles pages to corresponding GitHub issue

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#1083]
E.g. from http://obofoundry.org//principles/fp-009-users.html to the relevant ticket here: https://github.com/OBOFoundry/OBOFoundry.github.io/labels/principles

**Comment by nataled** [comment]
Good idea, though I think there is potential for confusion. You are suggesting that the principles pages be linked to the ticket for automated validation, but that is not always the appropriate destination if, for example, a user wishes to suggest a change to the principle itself. 

The principles currently have links to the automated check procedures (via the hyperlinked text "This check is automatically validated"). I suggest that we use those intermediate pages for the link you suggest. So, for example, the page http://obofoundry.org//principles/fp-009-users.html (already) links to http://obofoundry.org//principles/checks/fp_009, and the latter should then link to the appropriate ticket. 

As for direct links between the principles and the tickets about them, I'm a bit hesitant to make them, only because it seems we'd have to keep them open forever...

**Comment by cmungall** [comment]
I suppose we want to encourage comments on both the implementation and on the principle itself. I don't think a forever-open ticket on the latter is necessarily bad.

Or we could have have a template for a new issue pertaining to a principle, and include prominent link from the principles pages to a new issue with that template. Maybe the easiest, and avoids big confusing unthreaded discussions?

The main meta-principle here is openness of our principles, and including community input

**Comment by yongqunh** [comment]
Is it possible to add the eXtensible ontology development (XOD) principles, as defined in the paper:
https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-017-0169-2
These principles are more like implementation principles.

Oliver

**Comment by nataled** [comment]
Just discovered that we can create customized templates without 'saving' them (thus preventing a long list of issue templates). I propose adding a section to each principle, at the bottom (under Criteria for review) called "Feedback". I propose the following text, each linked to an appropriate template:

For questions, concerns, or suggestions pertaining to this Principle, please [create an issue on GitHub]().
For questions, concerns, or suggestions pertaining to the automated validation of this Principle, please [create an issue on GitHub]().

The part between the brackets would be linked to a template of the form https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?labels=attn%3A+Editorial+WG,principles&title=Principle+%23%3CENTER+NUMBER+HERE%3E+%22%3CENTER+NAME+HERE%3E%22+%3CENTER+ISSUE+TITLE%3E or https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?labels=attn%3A+Technical+WG,automated+validation+of+principles&title=Principle+%23%3CENTER+NUMBER+HERE%3E+%22%3CENTER+NAME+HERE%3E%22+-+automated+validation+%3CENTER+ISSUE+TITLE%3E as appropriate.

**Comment by nlharris** [comment]
Not sure what you mean about not "saving" customized templates, but this sounds like it would be helpful!

**Comment by nataled** [comment]
@nlharris If you go to the issue tracker page and click on New Issue, you'll come to a page with three pre-defined templates. Those are what I called 'saved' templates. What I noticed is that if you click on one, nearly all the pertinent information that would be needed is in the URI itself, including labels, assignments, title (maybe others, but we don't need more than that). Thus, templates can be made on-the-fly just using the URI :)
